src/App.test.js

```diff
@@ -1,8 +1,8 @@
-import { render, screen } from '@testing-library/react';
-import App from './App';
+// import { render, screen } from '@testing-library/react';
```

Comment:
We can delete these tests.

Reply:
True, but deleting them would also be separate from this PR, I think 😆

Reply:
Yes :D